setup.py

```diff
@@ -53,7 +53,7 @@ def run(self, *args, **kwargs):
     maintainer_email = 'h.gaiser@fizyr.com',
     cmdclass         = {'build_ext': BuildExtension},
     packages         = setuptools.find_packages(),
-    install_requires = ['keras', 'keras-resnet==0.1.0', 'six', 'scipy', 'cython', 'Pillow', 'opencv-python', 'progressbar2'],
+    install_requires = ['keras', 'keras-resnet==0.2.0', 'six', 'scipy', 'cython', 'Pillow', 'opencv-python', 'progressbar2'],
```

Comment:
Is this required? 0.2.0 has some changes that I don't agree with: https://github.com/broadinstitute/keras-resnet/issues/57

Reply:
At least 0.1.0 was not working for me. I will check this once more.

Reply:
Using resnet 0.1.0 I got this error. Did not investigate this error further since 0.2.0 was working
```
TypeError: in user code:
    /usr/lib/python3.8/site-packages/keras_resnet/layers/_batch_normalization.py:17 call  *
        return super(BatchNormalization, self).call(training=(not self.freeze), *args, **kwargs)
    TypeError: type object got multiple values for keyword argument 'training'
```

Reply:
Can you show the rest of that traceback? I'm curious which layer it is that gave that error.

Reply:
sure
```
Traceback (most recent call last):
  File "/home/cboden/retina_test/test.py", line 4, in <module>
    model = load_model("inference.h5")
  File "/usr/lib/python3.8/site-packages/keras_retinanet/models/__init__.py", line 87, in load_model
    return keras.models.load_model(filepath, custom_objects=backbone(backbone_name).custom_objects)
  File "/usr/lib/python3.8/site-packages/tensorflow/python/keras/saving/save.py", line 184, in load_model
    return hdf5_format.load_model_from_hdf5(filepath, custom_objects, compile)
  File "/usr/lib/python3.8/site-packages/tensorflow/python/keras/saving/hdf5_format.py", line 177, in load_model_from_hdf5
    model = model_config_lib.model_from_config(model_config,
  File "/usr/lib/python3.8/site-packages/tensorflow/python/keras/saving/model_config.py", line 55, in model_from_config
    return deserialize(config, custom_objects=custom_objects)
  File "/usr/lib/python3.8/site-packages/tensorflow/python/keras/layers/serialization.py", line 105, in deserialize
    return deserialize_keras_object(
  File "/usr/lib/python3.8/site-packages/tensorflow/python/keras/utils/generic_utils.py", line 369, in deserialize_keras_object
    return cls.from_config(
  File "/usr/lib/python3.8/site-packages/tensorflow/python/keras/engine/network.py", line 986, in from_config
    input_tensors, output_tensors, created_layers = reconstruct_from_config(
  File "/usr/lib/python3.8/site-packages/tensorflow/python/keras/engine/network.py", line 2029, in reconstruct_from_config
    process_node(layer, node_data)
  File "/usr/lib/python3.8/site-packages/tensorflow/python/keras/engine/network.py", line 1977, in process_node
    output_tensors = layer(input_tensors, **kwargs)
  File "/usr/lib/python3.8/site-packages/tensorflow/python/keras/engine/base_layer.py", line 922, in __call__
    outputs = call_fn(cast_inputs, *args, **kwargs)
  File "/usr/lib/python3.8/site-packages/tensorflow/python/autograph/impl/api.py", line 265, in wrapper
    raise e.ag_error_metadata.to_exception(e)
TypeError: in user code:

    /usr/lib/python3.8/site-packages/keras_resnet/layers/_batch_normalization.py:17 call  *
        return super(BatchNormalization, self).call(training=(not self.freeze), *args, **kwargs)

    TypeError: type object got multiple values for keyword argument 'training'


Process finished with exit code 1
```

Reply:
Ah I see, that doesn't make it much clearer unfortunately ^^

I'll see if I can have a look at it on Thursday. I would prefer not to move to 0.2.0 :)

Reply:
Hmm running :+1:
```
keras_retinanet.models.load_model('snapshots/resnet50_coco_best_v2.1.0.h5')
```

Gave no issues for me. Are you doing something else?

Reply:
Can you share the tf version you testet this? Got this crash with tf 2.2 and 2.3

Reply:
Also I am loading an inference 

> I think that's because tensorflow / keras started identifying the `self.anchors` as weights that it must save, which it shouldn't. Maybe there is a better workaround, but I have done this in a different project to circumvent this issue:
> [fizyr/tf-retinanet@d86be75](https://github.com/fizyr/tf-retinanet/commit/d86be7591b7aa2d17ddb1fb7025aa709b4e441f0)

Ok I will add this to the PR. Not sure about the implications of that workaround

Reply:
Hmm I've looked into it a bit and it seems keras now also passes a `training` argument, which is poorly handled by keras-resnet==0.1.0. In keras-resnet==0.2.0 this is changed to "update" the kwarg if the layer should be frozen. I still don't prefer to move to 0.2.0, but I guess it's the lesser of two evils :p